ultradns-rest/src/main/java/denominator/ultradns/model/ZoneList.java

```diff
@@ -0,0 +1,42 @@
+package denominator.ultradns.model;
```

Comment:
Yes we ran some E2E test scenario. 
REST method responses being casted to these objects by Feign Framework itself